src/components/EmptyState/EmptyState.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { render } from "@testing-library/react";
+import React from "react";
+
+import EmptyState from "./EmptyState";
+
+describe("EmptyState ", () => {
+  it("renders the title", () => {
+    const { container } = render(
+      <EmptyState title="Test title" image={<img alt="" src="" />} />
+    );
+    expect(container).toContainHTML("Test title");
```

Comment:
This could be slightly improve to `expect(screen.getByRole("heading", {name: "Test title"})).toBeInTheDocument();`